integration-tests/integration.test.js

```diff
@@ -1,12 +1,11 @@
 const { getByText, fireEvent, waitFor } = require('@testing-library/dom');
 const { startApp } = require('./test-app');
 
-describe('Tram-One', () => {
-	beforeEach(() => {
-		// clean up any tram-one properties between tests
-		window['tram-space'] = undefined;
-	});
```

Comment:
TLDR, I didn't notice this existed, ended up wasting a lot of time trying to reason around the global 'tram-space' in the other test files, and ended up making this clean up part of the `startApp` method.